helm-chart/values.yaml

```diff
@@ -222,6 +222,57 @@ cvat:
           protocol: TCP
           name: http
 
+  kvrocks:
+    enabled: true
+    external:
+      host: 127.0.0.1
+    existingSecret: "cvat-kvrocks-secret"
+    secret:
+      create: true
+      name: cvat-kvrocks-secret
+      password: cvat_kvrocks
+    replicas: 1
+    image: apache/kvrocks
+    tag: 2.7.0
+    imagepullploicy: IfNotPresent
+    labels: {}
+    #  test: test
+    annotations: {}
+    # test.io/test: test
+    resources: {}
+    affinity: {}
+    tolerations: []
+    # nodeAffinity:
+    #   requiredDuringSchedulingIgnoredDuringExecution:
+    #     nodeSelectorTerms:
+    #     - matchExpressions:
+    #       - key: kubernetes.io/e2e-az-name
+    #         operator: In
+    #         values:
+    #         - e2e-az1
+    #         - e2e-az2
+    additionalEnv: []
+    # Example:
+    #  - name: volume-from-secret
+    # - name: TEST
+    #  value: "test"
```

Comment:
The examples are misaligned. Also, the first example seems strange? There's a name, but no value, and the name doesn't feel like an environment variable name.